deb/debian/stackdriver-agent.init

```diff
@@ -68,7 +68,17 @@ find_libjvm_so () {
     echo $java_lib_dir
 }
 
-[ -n "$JAVA_LIB_DIR" ] || JAVA_LIB_DIR=$(find_libjvm_so)
+find_libjvm_so_jdk () {
+    local java_lib_dir
+    local java_home=$(dirname "$(readlink -f "$(bash -lc "which java 2>/dev/null")")")/..
+    if [ -f "$java_home/lib/server/libjvm.so" ]; then
```

Comment:
This could also be accomplished by simply adding `"lib/server"` to the `lib_subdirs` array in `find_libjvm_so`, can't it?

Reply:
As much for me, I've been fooled by
```
for subdir in "${lib_subdirs[@]}"; do
        lib_subdirs_with_jre+=("$subdir" "jre/$subdir")
```